features/request/request.feature

```diff
@@ -328,7 +328,30 @@ Feature: Formatting requests for sending to the ID Broker API
         {
           "employee_id": "12345",
           "type": "webauthn",
-          "label": "Blue security key"
+          "label": "Blue security key",
+          "recovery_email": ""
```

Comment:
This seems unexpected: if no "recovery_email" parameter was specified in the `Given/And` steps, then it seems like the body shouldn't have a `recovery_email` entry. What's the reasoning for wanting that to be present in the request body if no `recovery_email` was provided?